firmware/config/stm32f4ems/efifeatures.h

```diff
@@ -231,7 +231,7 @@
  * Do we need file logging (like SD card) logic?
  */
 #ifndef EFI_FILE_LOGGING
-#define EFI_FILE_LOGGING TRUE
+#define EFI_FILE_LOGGING FALSE
```

Comment:
we can't turn this off either

Reply:
We need to turn it for MRE at leads. Until we have SD card on MRE. )